src/language-service/src/configuration.ts

```diff
@@ -54,6 +54,10 @@ export class ConfigurationService implements IConfigurationService {
     if (!this.token && process.env.HASS_TOKEN) {
       this.token = process.env.HASS_TOKEN;
     }
+    if (!this.url && !this.token && process.env.SUPERVISOR_TOKEN) {
```

Comment:
This is what the above HASS_SERVER & HASS_TOKEN originally tried to solve.

That said, not against this addition as it allows a little clean up on the add-on part.

../Frenck

Reply:
@frenck, only the VS Code add-on exposes `HASS_SERVER` and `HASS_TOKEN`. This change is mainly for other add-ons, like _Advanced Terminal_.

Reply:
Yeah I get that, but I can clean that part up after this one 😉

Reply:
Exactly. Great.